include/deal.II/numerics/vector_tools.h

```diff
@@ -1845,6 +1845,10 @@ namespace VectorTools
    * It is assumed that the number of components of the function @p
    * exact_solution matches that of the finite element used by @p dof.
    *
+   * To compute a global error norm of a finite element solution, use
+   * error_norm_from_cell_errors() with the Vector computed with this
```

Comment:
maybe qualify the name with the namespace, to make it even clearer?

Vector -> output vector